gwy/templates/go.mod.tmpl

```diff
@@ -9,6 +9,6 @@ require (
 	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
-	google.golang.org/grpc v1.34.0
-	google.golang.org/protobuf v1.24.0
+	google.golang.org/grpc v1.35.0
```

Comment:
I would upgrade to the latest grpc version here (1.42.X)

Reply:
> I would upgrade to the latest grpc version here (1.42.X)

Hmm, looks like I missed that one!